setup/ci/jenkins.md

```diff
@@ -91,56 +91,73 @@ b. Under __Crumb Algorithm__, select __Default Crumb Issuer__.
 
 ![](/setup/ci/jenkins_enable_csrf.png)
 
-## Jenkins script execution stage
+## Configure script stage
 
 ### Purpose
-The script stage lets a Spinnaker user run an arbitrary shell, python, or
-groovy script on a Jenkins instance as a first class stage in Spinnaker.
-This is good for launching an integration/functional test battery
+The [Script stage](/reference/pipeline/stages/#script) lets you run an arbitrary
+shell, Python, or Groovy script on a Jenkins instance as a first class stage in
+Spinnaker. This is good for launching an integration/functional test battery
```

Comment:
Really minor (and pre-existing), but the term "battery" sticks out to me here...maybe "test suite" instead?  (I've never seen the word "battery" in the sense of "many similar things" except in the form "battery of", ex "battery of tests").